### DISCUSSION TITLE
1.0.0

### DISCUSSION THREAD

**Discussion by buchdag** [nginx-proxy/nginx-proxy#1919] [Announcements]
## What's Changed
* [Path based routing](https://github.com/nginx-proxy/nginx-proxy#path-based-routing) (`VIRTUAL_PATH`) by @buchdag in https://github.com/nginx-proxy/nginx-proxy/pull/1901
* Bump docker-gen to `0.8.2` by @buchdag in https://github.com/nginx-proxy/nginx-proxy/pull/1902
* chore(deps): bump golang from `1.16.7` to `1.17.8` by @dependabot in https://github.com/nginx-proxy/nginx-proxy/pull/1884 and  https://github.com/nginx-proxy/nginx-proxy/pull/1917
* the image is now also available from GitHub Container Registry : [`ghcr.io/nginx-proxy/nginx-proxy`](https://ghcr.io/nginx-proxy/nginx-proxy)

## Contributors
Many thanks to the following people for their work and/or feedback on the path based routing feature.
@AlexanderLieret @junderw @warren-ru @JMLX42 @OliverRhyme @trowj @WolfspiritM @rodrigoaguilera @RafaelKr @gregsymons

**Full Changelog**: https://github.com/nginx-proxy/nginx-proxy/compare/0.10.1...1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/nginx-proxy/nginx-proxy/releases/tag/1.0.0'>1.0.0</a>.</em>

**Comment by RafaelKr** [comment]
Awesome mate! Thank you very much for maintaining this project and making this long awaited feature reality! :tada:
